### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by benjub**
rewrap README.TXT; all files: remove trailing spaces, remove double blank lines, remove blank lines before closing parentheses/braces/brackets

### CODE REVIEW

**Comment by digama0** [comment]
oh wow, there are exact byte counts in the printout. That is very brittle to changes! We should think about redacting that somehow in the test runner (or just removing it from the printout, I don't think it's a particularly useful measure for users).

**Comment by benjub** [comment]
> oh wow, there are exact byte counts in the printout. That is very brittle to changes! We should think about redacting that somehow in the test runner (or just removing it from the printout, I don't think it's a particularly useful measure for users).

I do not know. Having it might still avoid some subtle problems. And the option `--bless` (which I have generally seen called `--promote` in other projects) is handy.

File: tests/run_test.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ if [ "$1" = "--help" ]; then usage; exit; fi
 # Allow overriding the 'metamath' command using the METAMATH env variable
 cmd="${METAMATH:-metamath}"
 
-# Alternatively, use the '-c metamath' argument which overrides the env variable
+# Alternately, use the '-c metamath' argument which overrides the env variable
```
Comment (digama0) [inline tests/run_test.sh:35]:
```suggestion
# Alternatively, use the '-c metamath' argument which overrides the env variable
```
I believe the original was correct. "Alternatively" is used to give alternatives in the sense "you can do this, or alternatively you can do that", while "Alternately" is used for something that goes back and forth between values, as in "the lines are alternately long and short".

**Review by digama0 — APPROVED** [review]

File: tests/run_test.sh (hunk 1 above)
Comment (benjub) [reply tests/run_test.sh:35]: You're right, I'll revert the change.

File: src/mmveri.c [hunk 2]
```diff
@@ -354,7 +349,7 @@ nmbrString *assignVar(nmbrString *bigSubstSchemeAss,
   v = -1; /* Position in bigSubstSchemeVars */
   p = 0; /* Position in bigSubstSchemeAss */
   q = 0; /* Position in bigSubstInstAss */
- ambiguityCheck: /* Re-entry point to see if unification is unique */
+  ambiguityCheck: /* Re-entry point to see if unification is unique */
```
[the remark this answers is not in the document]
Comment (benjub) [reply src/mmveri.c:352]: Completely unindent labels: https://softwareengineering.stackexchange.com/questions/151806/indenting-labels-in-c

[the remark this answers is not in the document]
Comment (digama0) [reply src/mmveri.c:352]: That works for me.

**Pull request merged**